pkg/report/report.go

```diff
@@ -240,3 +240,26 @@ func getChecks(checkResults []rule.CheckResult, opts *ReportOptions) []Check {
 	}
 	return checks
 }
+
+// DiscardCheckResultsBelowMinStatus removes all checks of the report, whose statuses are lower than the specified one.
+func (r *Report) DiscardCheckResultsBelowMinStatus(minStatus rule.Status) {
```

Comment:
```suggestion
// SetMinStatus sets minStatus of the report. It also removes all checks with status less than the specified one.
// If the new minStatus is less that the original one, nothing is changed.
func (r *Report) SetMinStatus(minStatus rule.Status) {
```

I think this would be a better name for the method. WDYT?